src/lib.rs

```diff
@@ -99,6 +90,49 @@ impl ConditionallySelectable for AffinePoint {
     }
 }
 
+impl AffinePoint {
+    /// Get a fixed chosen point, that has a small x-coordinate, and its corresponding y-coordinate.
+    /// The point is then reduced according to the prime field. We need only to
+    /// state the cooridnates, so users can exploit its properties
+    /// which are proven by tests, to prove general arithemtic.
+    ///
+    ///
+    /// Using: x = 1539098E9CBCC1D50CCC77B0E1804E8D6EEF947A6FD0FB2CA3D063F54E10DDE9
+    ///        y = 6540D21E7007DC603B0D848E832A862FB53BB87E05DA8257CD482CC3FD6FF4D
+    pub fn fixed_point() -> Self {
+        let x_bytes: [u8; 32] = [
+            233, 221, 16, 78, 245, 99, 208, 163, 44, 251, 208, 111, 122, 148, 239, 110, 141, 78,
+            128, 225, 176, 119, 204, 12, 213, 193, 188, 156, 142, 9, 57, 21,
+        ];
+
+        let y_bytes: [u8; 32] = [
+            77, 255, 214, 63, 204, 130, 212, 124, 37, 168, 93, 224, 135, 187, 83, 251, 98, 168, 50,
+            232, 72, 216, 176, 3, 198, 125, 0, 231, 33, 13, 84, 6,
+        ];
+
+        AffinePoint {
+            x: Fq::from_bytes(&x_bytes).unwrap(),
+            y: Fq::from_bytes(&y_bytes).unwrap(),
+        }
+    }
+
+    /// Loops and creates new generators until a proper one is found.
+    pub fn generator() -> AffinePoint {
+        let mut random_bytes = [0; 32];
+        loop {
+            let generator = AffinePoint::from_bytes(random_bytes);
+            if generator.is_some().unwrap_u8() == 1 {
+                let generator = generator.unwrap();
+                if generator.is_prime_order().unwrap_u8() == 1 {
+                    return generator;
+                }
+            }
+
+            random_bytes[0] += 1;
+        }
+    }
+}
+
```

Comment:
Why is this function needed?

We can just work with the same generator point always, I don't see the point of having this.